services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/internal/operations/ScanOperation.java

```diff
@@ -56,6 +56,12 @@ private ScanOperation(ScanEnhancedRequest request) {
         return new ScanOperation<>(request);
     }
 
+    private static boolean isAttributeToProjectAnExpression(final String attr,
```

Comment:
Maybe consider extracting to a utility class

Reply:
Closing the Pull request without merge as will use other user friendly approach.